api/server.ts

```diff
@@ -5,8 +5,33 @@ const app = express();
 app.use(express.json());
 app.use(cors());
 
-app.post('/', (req, res) => {
-  res.send(req.body.test || 'Hello World');
+app.post('/professor', (req, res) => {
+  // API returns single professor data or null if doesn't exist
+  //I'm using a single object, but Ideally this should be a nested dictionary? this is the format that I'll just stick with 
+  let professorReturn = {
```

Comment:
Multiple ESLint errors come from the lack of spacing in comments and using `let` over `const`. Install the ESLint extension and commit changes before this PR can be accepted.